### PR TITLE
G2 2023 v0 issue#12960

### PR DESCRIPTION
**Pull request by b21hamwe**
issue#12965 from yesterday follows here as well since an un-noted problem with github desktop occured. 
Not the best solution, but issue-responsible (b21kurar) approved of this solution.

For issue#12960
I would still recommend the testing individual to look out for redundant variables in the diagram model, although i've tried my very best for this not to occur.

### CODE REVIEW

File: backend-models/codeviewerService.php.svg
Comment (a21jeaha) [inline backend-models/codeviewerService.php.svg:1]: some duplicates and missing variables that needs fixing :)

dubbles
-------

sectionName
courseId x 2
exampleName 
currentId
word
wordList


missing
-------

codeDir
box

**Review by a21jeaha — CHANGES_REQUESTED** [review]
variables need adjusting, iv supplied a list of found issues :) 

dubbles
-------

sectionName
courseId x 2
exampleName 
currentId
word
wordList


missing
-------

codeDir
box

**Comment by a19hakja** [comment]
Good job, looks good to me aswell. Merging.

**Comment by b21kurar** [comment]
Messy merging, let's not do this again

**Pull request merged**